packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/common/get-transaction-receipt-by-tx-id.ts

```diff
@@ -0,0 +1,203 @@
+import { LoggerProvider, LogLevelDesc } from "@hyperledger/cactus-common";
```

Comment:
@jscode017 Is this PR dependent on the other one with the get tx receipt PR? If yes, please mark the dependency and we can continue reviewing this once that one was merged (so that the diff is smaller, easier to process)